AppCenterDataStorage/AppCenterDataStorage/MSDataStore.m

```diff
@@ -186,6 +186,18 @@ + (void)deleteDocumentWithPartition:(NSString *)partition
                                           completionHandler:completionHandler];
 }
 
++ (void)setOfflineMode:(BOOL)offlineMode {
+  @synchronized([MSDataStore sharedInstance]) {
+    [MSDataStore sharedInstance].offlineMode = offlineMode;
+  }
+}
+
++ (BOOL)isOfflineMode {
+  @synchronized([MSDataStore sharedInstance]) {
+    return [MSDataStore sharedInstance].offlineMode;
```

Comment:
This does not need to be synchronized since the property is `atomic`

Reply:
Yep. It was changed from `nonatomic` to `atomic` and forgot to clean up those.